index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>demosite</title>
+    <meta name="description " content="demosite">
+
+    <!-- CSS -->
+    <link rel="stylesheet" href="https://unpkg.com/ress/dist/ress.min.css">
```

Comment:
```suggestion
    <link rel="stylesheet preload" as="style" href="https://unpkg.com/ress/dist/ress.min.css">
```
preloadつけたほうがサイトの表示早いです。